apps/site/util/downloadUtils/simple.tsx

```diff
@@ -0,0 +1,247 @@
+          version: versionWithPrefix,
+          kind: 'installer',
+        }),
+        version: versionWithPrefix,
+        release: {
+          source: getNodeDownloadUrl({
+            version: versionWithPrefix,
+            kind: 'source',
+          }),
+          changelog: `${BASE_CHANGELOG_URL}${minor.version}`,
+          blogPost: `${RELEASE_POST_URL}${versionWithPrefix}`,
+        },
+      };
+    }),
+});
+
+export const buildMetaBarItems = (
+  {
+    codename,
+    status,
+    currentStart,
+    releaseDate,
+    minorVersions,
+    modules,
+    npm,
+    v8,
+  }: NodeRelease,
+  t: (key: string) => string
+) => ({
+  ...(codename && {
+    [t('layouts.simpleDownload.codename')]: codename,
+  }),
+  [t('layouts.simpleDownload.status')]: t(
+    `layouts.simpleDownload.statusNames.${status}`
+  ),
+  [t('layouts.simpleDownload.firstReleased')]: (
+    <FormattedTime date={currentStart} />
+  ),
+  [t('layouts.simpleDownload.lastUpdated')]: (
+    <FormattedTime date={releaseDate} />
+  ),
+  [t('layouts.simpleDownload.minorVersions')]: minorVersions.length,
+  [t('layouts.simpleDownload.nApiVersion')]: modules,
+  [t('layouts.simpleDownload.npmVersion')]: npm,
+  [t('layouts.simpleDownload.v8Version')]: v8,
+});
+
+type SidebarGroup = ComponentProps<typeof ProgressionSidebarGroup>;
+
+export const groupReleasesByStatus = (
+  releaseData: Awaited<ReturnType<typeof getReleaseData>>,
+  pathname: string
+): Array<SidebarGroup> => {
+  let simplified = false;
+
+  // Reduce the release data into a record grouped by release status (e.g., 'LTS', 'Current')
+  const grouped = releaseData.reduce<Record<string, SidebarGroup>>(
+    (acc, release) => {
+      const statusKey = release.status;
+
+      // Initialize the group if it doesn't exist yet
+      if (!acc[statusKey]) {
+        acc[statusKey] = {
+          groupName: statusKey,
+          items: [],
+        };
+
+        // Check if the current pathname indicates a simplified download page
+        if (statusKey === 'LTS' && pathname.endsWith('simplified')) {
+          simplified = true;
+        }
+      }
+
+      // Build the label: always include major version, optionally codename
+      const labelParts = [`v${release.major}`];
+      if (release.codename) {
+        labelParts.push(release.codename);
+      }
+
+      // Add the release to the group's items
+      if (simplified) {
+        acc[statusKey].items.push({
+          label: labelParts.join(' '),
+          link: `/download/simplified`,
+        });
+
+        simplified = false;
+      } else {
+        acc[statusKey].items.push({
+          label: labelParts.join(' '),
+          link: `/download/${release.major}`,
+        });
+      }
+
+      return acc;
+    },
+    {}
+  );
+
+  // Return the grouped items as an array for sidebar consumption
+  return Object.values(grouped);
+};
+
+export const extractVersionFromPath = (
+  pathname: string | undefined
+): string | null => {
```

Comment:
```suggestion
): number => {
```

If we have this return a number / NaN, we can avoid the `Number(xyz)` call when comparing the major.